tests/jq

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
This file is janky, but it was my best solution to get tests running. I'm happy to have a conversation about it if this comment is insufficient.

Reply:
This has been removed